From a6fdbfd7d82df63c49dcc01b98d25110c0ff4caf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 21 Dec 2022 10:36:42 +0100
Subject: [PATCH] Stop installing documentation files to top-level
 site-packages

Add `format = sdist` to `include` table values to prevent
the documentation files from being installed directly into
site-packages, i.e.:

    /usr/lib/python3.11/site-packages/CHANGELOG.md
    /usr/lib/python3.11/site-packages/LICENSE

Originally reported by Anna Vyalkova on https://bugs.gentoo.org/887569.
---
 pyproject.toml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index eaeafb90..b0608a58 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,9 +34,9 @@ classifiers = [
 urls = { Changelog = "https://github.com/pycqa/isort/blob/main/CHANGELOG.md" }
 include = [
     { path = "tests", format = "sdist" },
-    { path = "ACKNOWLEDGEMENTS.md" },
-    { path = "CHANGELOG.md" },
-    { path = "LICENSE" },
+    { path = "ACKNOWLEDGEMENTS.md", format = "sdist" },
+    { path = "CHANGELOG.md", format = "sdist" },
+    { path = "LICENSE", format = "sdist" },
 ]
 
 [tool.poetry.dependencies]
